google/api_core/client_logging.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import json
+import os
+
+from typing import List, Optional
+
+_LOGGING_INITIALIZED = False
+_BASE_LOGGER_NAME = "google"
+
+# Fields to be included in the StructuredLogFormatter.
+#
+# TODO(https://github.com/googleapis/python-api-core/issues/761): Update this list to support additional logging fields.
+_recognized_logging_fields = [
+    "httpRequest",
+    "rpcName",
+    "serviceName",
```

Comment:
I just want to confirm that we are ok with deviating from the standard snake case that's typically used

https://peps.python.org/pep-0008/#function-and-variable-names

eg.

```suggestion
    "http_request",
    "rpc_name",
    "service_name",
```

Reply:
Yes it is fine and in fact a requirement in some cases to have camel case names.